conf.d/main

```diff
@@ -6,19 +6,8 @@ LDAP_DOMAIN=example.com
 SRC=/usr/local/src
 WEBROOT=/var/www/phpldapadmin
```

Comment:
I would suggest changing the EV for the webroot instead of hard coding the full file path into the code like how it was previously done. Otherwise, I'd remove this line entirely if we are going the hardcoded path route.

Reply:
Good catch Mattie! I'll just remove this line for now.